### ISSUE TITLE
The Input Choiceset type for an adaptive card renders invisible

### ISSUE THREAD

**Issue by AyoOlubeko** [Microsoft/BotFramework-Emulator#206]
When using the input choiceset type adaptive card (http://adaptivecards.io/explorer/#InputChoiceSet), the checkboxes that are rendered in the emulator are invisible and do not show up.

**Comment by eanders-MS** [comment]
Hi @AyoOlubeko I'd like to try to repro this. Could you please post the json of the adaptive card message?

**Comment by AyoOlubeko** [comment]
Sure thing, relevant code snippet I used below. When you try it out in the emulator, there is no outline of a checkbox when it is rendered, so you'll have to click around until you find where the checkbox is.

```
var productSelection = new builder.Message(session)
    .addAttachment({
        contentType: "application/vnd.microsoft.card.adaptive",
        content:
        {
            type: "AdaptiveCard",
            body:
            [
                {
                    "type": "TextBlock",
                    "text": "Select an Azure product"
                },
                {
                    "type": "Input.ChoiceSet",
                    "id": "productSelection",
                    "isMultiSelect": true,
                    "style": "compact",
                    "choices": 
                    [
                        {
                          "title": "Azure SQL database",
                          "value": "Azure SQL database"
                        },
                        {
                            "title": "Azure webapp",
                            "value": "Azure webapp"
                        },
                        {
                            "title": "Azure botservice",
                            "value": "Azure botservice"
                        },
                        {
                            "title": "Azure VM",
                            "value": "Azure VM"
                        },
                        {
                            "title": "Azure IOT Hub",
                            "value": "Azure IOT Hub"
                        },
                        {
                            "title": "Other",
                            "value": "Other"
                        }
                    ]
                }           
            ],
            actions:
            [
                {
                    "type": "Action.Submit",
                    "title": "Submit",
                    "data": { "message" : "productSelection"}  
                }
            ]
        }
    });

session.send(productSelection);
```

**Comment by eanders-MS** [comment]
@AyoOlubeko, Thanks for the bug report. I've made a quick fix in CSS. It will be available in the next release.

Side note: Some unintended style bleed is happening here. The emulator's CSS should not affect WebChat's styling. I'll take this as a separate issue.

**Comment by eanders-MS** [comment]
Fixed in emulator v3.5.31
